app/components/finance/npq/vat_payment_row.html.erb

```diff
@@ -0,0 +1,6 @@
+<tr class="govuk-table__row">
+  <td scope="row" class="govuk-table__cell govuk-!-width-one-quarter"><%= t("finance.vat") %></td>
+  <td scope="row" class="govuk-table__cell govuk-table__cell--numeric govuk-!-width-one-quarter"/>
```

Comment:
same thing but we can do 2 halves if the 2 of the quarters are empty?

Reply:
It doesn't work, the row collapses